packages/marko-web/components/element/picture.marko

```diff
@@ -0,0 +1,25 @@
+import { getAsArray } from "@parameter1/base-cms-object-path";
+
+$ const hasImage = Boolean(input.image.src);
+
+<if(hasImage)>
+  <if(input.link)>
+    <marko-core-link ...input.link>
+      $ input.link = false;
```

Comment:
This should be unnecessary if defining the props manually. If not, use spreading to separate the attributes, rather than overwriting them.
```marko
$ const { link, ...props } = input;
<if(link)>
  <marko-web-picture ...props />
```